e2e_playwright/st_both_query_params_error.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+
+st.query_params.y = 1.23
+
+# Should throw an error after both apis are used
+params = st.experimental_get_query_params()
```

Comment:
added an e2e as that seems faster than writing all the python unit tests, especially since we plan to remove support for `experimental_get_query_params` at some point.